src/celengine/galaxy.cpp

```diff
@@ -602,7 +602,7 @@ GalacticForm* buildGalacticForms(const fs::path& filename)
     height = img->getHeight();
     rgb    = img->getComponents();
 
-    auto& rng = getRNG();
+    celmath::Jsf32 rng(UINT32_C(0xa0030c29));
```

Comment:
magic constants, we all like them! :)))

could you provide names for them with some descriptive comments?